### PR TITLE
Show available langs directly in the document pages

### PR DESCRIPTION
**Pull request by asaunier**
Related to https://github.com/c2corg/v6_ui/issues/1680

The layout of the lang links are still a bit raw but I have not much inspiration to make them nicer (patch/help welcome :P ). The available langs entry in the Plus menu has been removed.

@Courgetteandratatouille this might impact your work about the page layout redesign.

capture d ecran 2017-08-10 a 22 16 37

### CODE REVIEW

**Comment by asaunier** [comment]
@brunobesson @Courgetteandratatouille Looks good to you?

File: less/viewdetails.less [hunk 1]
```diff
@@ -606,6 +606,10 @@
       color: @C2C-orange
     }
   }
+
```
Comment (stef74) [inline less/viewdetails.less]: Why a blank line ?

Comment (asaunier) [inline less/viewdetails.less]: To be consistent with the other blocks? Except maybe ``.area-associations`` just above, there are blank lines before selector blocks.

**Comment by asaunier** [comment]
@stef74 Thanks for the review. Merging :)

**Pull request merged**